packages/eslint-plugin/package.json

```diff
@@ -49,6 +49,7 @@
     "@typescript-eslint/utils": "5.40.0",
     "debug": "^4.3.4",
     "ignore": "^5.2.0",
+    "natural-compare-lite": "^1.4.0",
```

Comment:
the thing i like about this package is how the readme is 60% of the package's bytes

![image](https://user-images.githubusercontent.com/7462525/197425417-7d174bd5-9968-44a5-a20e-75194960e4c3.png)

Reply:
Hey,
thanks for the patch!
However I think this package is unnecessary, indeed node supports this natively for a very long time. See this example in node's REPL (here it's node version 10...):
```
> ["a2", "a10", "a1"].sort((a, b) => a.localeCompare(b, undefined, { numeric: true }))
[ 'a1', 'a2', 'a10' ]
```

I think that as an ecosystem we should avoid depending on too many small packages.

Thanks!